sdk/include/psdk/shdeprecated.idl

```diff
@@ -343,17 +343,17 @@ typedef struct basebrowserdataxp
     DWORD               _fCreatingViewWindow;
     UINT                _uActivateState;
 
-    LPCITEMIDLIST       _pidlNewShellView;
+    PCIDLIST_ABSOLUTE       _pidlNewShellView;
```

Comment:
Here and at some other places, remove some middle-spaces to maintain v-alignment. Thanks.

Reply:
Okay, now I understand why the formatting error popped up. Thanks for pointing it out @SergeGautherie. I will fix it ASAP.

Reply:
No, the formatting error popped up because the thing is not configured correctly ;)

Reply:
Made changes to maintain v-alignment in the code.